src/cli/Microsoft.DotNet.xunit.performance.runner.cli/project.json

```diff
@@ -20,17 +20,18 @@
   },
 
   "frameworks": {
-    "dnxcore50": {
-      "imports": "portable-net45+win8",
+    "netstandard1.5": {
```

Comment:
Why are you using net standard here? You have `emitEntryPoint: true` so I assume this is an application?

`netcoreapp1.0` is the official TfM for this.